python/tk_houdini/bootstrap.py

```diff
@@ -87,7 +87,7 @@ def bootstrap_classic():
     except Exception as e:
         bootstrap_exception(
             "Toolkit bootstrap failed to extract the current context from the "
-            "environment! The SG integration will be disabled. Details: "
+            "environment! The PTR integration will be disabled. Details: "
```

Comment:
In the previous line, you can replace the word Toolkit by "Flow Production Tracking Toolkit" or just PTR as defined in our wiki.